configurator/src/.backupignore.template

```diff
@@ -0,0 +1 @@
+data/graph/mainnet/*
```

Comment:
If it's not going into a format string, it shouldn't be .template

Reply:
ok